main.js

```diff
@@ -0,0 +1,35 @@
+function generateNumbers() {
+    let numbersOnScreen = Math.floor(Math.random() * 3) + 3;
+    let allNumbers = [];
+
+    for (i = 0; i < numbersOnScreen; i++) {
+        let randNumber = Math.floor(Math.random() * 101);
+
+        allNumbers.push(randNumber);
+    }
+
+    return allNumbers;
+}
+
+
+function displayNumbers(allNumbers) {
+    let numbersOnScreen = allNumbers.length;
+
+    for (i = 0; i < numbersOnScreen; i++) {
+        let randNumber = allNumbers.pop();
+        let number = document.createElement('div');
+
+        number.textContent = randNumber;
+        number.setAttribute('class', 'numberBox');
+        document.querySelector('#play_area').appendChild(number);
+    }
+}
+
+let randomNumbers = generateNumbers();
+
+displayNumbers(randomNumbers);
```

Comment:
this generates and "displays" the random numbers on page load, shouldnt it be done when the game starts? not a big difference but feels like the right place to do it to me